test/WebSites/InProcessWebSite/Program.cs

```diff
@@ -22,5 +25,15 @@ public static void Main(string[] args)
 
             host.Run();
         }
+
+        private static void CheckEnvironmentVariables()
+        {
+            var envVariable = Environment.GetEnvironmentVariable("ASPNETCORE_INPROCESS_INITIAL_WRITE");
```

Comment:
Just write in the Main. Would it hurt other tests?

Reply:
We need to selectively write/not write based on the env variable. Whether we write first or not will change the behavior of what handle is stored by Console.

Reply:
I see